gitpod-io/website
# partial-fix: correct filepath for `Edit in Gitpod`

## Description

This PR Proposed to partially fixed #948. It would fix all the pages that are there in children. But, this doesn't seem to work for the parent's `index.md` file, because this logic doesn't seem to work => 

https://github.com/gitpod-io/website/blob/b21e0f4a72bd888043d62bfcaef9338d818e4b97/src/lib/components/docs/edit-in-gitpod.svelte#L7-L10

Any possible suggestions @Cahllagerfeld?

## Related Issue(s)

Related #948 

## How to test

- Open Preview
- Go to `/docs`
- Click or Hover on "Edit in Gitpod" button to check the URLs

## Release Notes



```release-note
NONE
```
